sw/device/lib/dif/dif_uart.h

```diff
@@ -0,0 +1,54 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef _DIF_UART_H_
+#define _DIF_UART_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+/**
+ * @brief TODO
+ *
+ * TODO
+ */
+typedef enum dif_uart_watermark {
+  lvl1 = 0,
+  lvl4,
+  lvl8,
+  lvl16,
+  lvl30,
+} dif_uart_watermark_e;
```

Comment:
These names are too short to be exported.  I think `dif_uart_watermark_level_1` is probably the name style here.  Also `_e` is only a verilog style thing, this should be `_t`.

Reply:
Good point, thank you

Reply:
I think maybe `dif_uart_watermark_char_n` would be more readable than `dif_uart_watermark_level_1`? As in 30 character watermark is probably more representative in terms of UART than level 30 watermark.